docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md

```diff
@@ -51,7 +51,7 @@ If you already have an EDMX file, you can skip this step.
 
 #### To create an EDMX file
 
-- If you don't already have an EDMX file, you can follow the instructions at the end of this walkthrough in the step **To configure the Entity Data Model**.
+- If you don't already have an EDMX file, you can follow the instructions at the end of this walkthrough in the step [Configuring the Entity Data Model](#configuring-the-entity-data-model).
```

Comment:
> `#configuring-the-entity-data-model`

This should be `generating-fsharp-types-from-edmx.md#configuring-the-entity-data-model.md`.